.github/workflows/ci.yml

```diff
@@ -166,9 +166,13 @@ jobs:
         path: old-versions
         key: ${{ runner.os }}-test-old-versions-1.12.0
         restore-keys: ${{ runner.os }}-test-old-versions-1.11.0
+    
+    - name: Make postgres sudoer
+      run: |
+        echo 'postgres ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
 
-    - name: Run Update Tests
-      run: su postgres -c 'sh tools/build -pg$PGVERSION -cargo-pgx /home/postgres/pgx/0.4/bin/cargo-pgx -cargo-pgx-old /home/postgres/pgx/0.2/bin/cargo-pgx test-updates 2>&1'
+    - name: Run binary update tests
+      run: OS_NAME=ubuntu OS_VERSION=20.04 su postgres -c 'sh tools/testbin -bindir .. -version 1.11.0 -dbroot /tmp/db -pgport 28814 deb'
```

Comment:
As I learned this afternoon, we actually run on Debian 10!

My near-term goal after finishing the current batch of release automation work is going to be extracting the rest of the Docker image building logic out of the packaging scripts and into the image-building script I pushed up today and (sadly) moving that script into the private repository so we can build arm64 images too, and then run both our CI here on that image and ALSO build packages on the same image.

Currently the packaging scripts build and discard a Docker image every time! :(

Reply:
I'm not sure testbin will work as is.  I think we need a third run mode (in addition to the existing 'deb' and 'rpm').

Reply:
I think this is what we need here:  https://github.com/timescale/timescaledb-toolkit/pull/556/files#diff-018192a8c8e7dea0df3dbbfdc766bfad7a3017167956b9714f8653b5f7644f72R111

Reply:
This should work:

```
su postgres -c 'OS_NAME=debian OS_VERSION=11 tools/testbin -version no -bindir / -dbroot /tmp/db -pgport 28800 ci 2>&1'
```

Although the chown bug-fix in PR #562 will need to go in first.

Reply:
Oops, I forgot to suggest using -pgversions - this is what we want:

```
# TODO We've always tested only one postgres version here.  We test them all at release
#  time anyway.  If we want to test them here, too, let's make it less slow first.
su postgres -c 'OS_NAME=debian OS_VERSION=11 tools/testbin -pgversions 14 -version no -bindir / -dbroot /tmp/db -pgport 28800 ci 2>&1'
```